### PR TITLE
Improve help messages

### PR DESCRIPTION
**Pull request by vicancy**
@vwxyzh  @chenkennt

### CODE REVIEW

File: src/docfx/HelpTextGenerator.cs [hunk 1]
```diff
@@ -17,19 +18,23 @@ static class HelpTextGenerator
             AddDashesToOption = true
         };
 
+        private static readonly string License;
+        private static readonly string GeneralUsage;
+        private static readonly string ProductName;
         static HelpTextGenerator()
```
Comment (vwxyzh) [inline src/docfx/HelpTextGenerator.cs]: add new line

File: src/docfx/Models/OptionUsageAttribute.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    using System;
+
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = true, Inherited = false)]
+    public class OptionUsageAttribute : Attribute
+    {
+        public string Name { get; }
+
+        public OptionUsageAttribute(string name)
+        {
+            this.Name = name;
```
Comment (vwxyzh) [inline src/docfx/Models/OptionUsageAttribute.cs]: remove this

Comment (vicancy) [inline src/docfx/Models/OptionUsageAttribute.cs]: updated

**Comment by chenkennt** [comment]
:+1:

**Pull request closed without merging**